content/ios/user.md

```diff
@@ -106,16 +96,31 @@ Sometimes users would want to change their passwords to secure their accounts. T
                  password:@"newPassword"
         completionHandler:^(SKYUserRecordID *user, NSError *error) {
     if (error) {
-        NSLog(@"Can't change password: %@", error);
+        NSLog(@"can't change password: %@", error);
         // Can be old password not matched?
         return;
     }
 
-    NSLog(@"Password change successfully");
+    NSLog(@"password change successfully");
 }]
 ```
 
-## Disocver users by emails
+# Current User
+## Getting current user id
+
+You can get the `currentUserRecordID` by:
+
+```obj-c
+if (container.currentUserRecordID) {
+    // user is logged in, proceed
+} else {
+    // ask user to sign up / login
+}
```

Comment:
@HenryHins This code snippet seems like telling the reader how to check whether a user is logged in instead of how to get the current user ID. How about something like `userID = container.currentUserRecordID`?